datafusion/src/physical_plan/sort.rs

```diff
@@ -201,15 +201,6 @@ fn sort_batch(
         None,
     )?;
 
-    let schema = Arc::new(Schema::new(
```

Comment:
the fix is to delete this code, the rest of the file is a test

Reply:
Nice and clean 🧹🧼